app/controllers/idv/verify_info_controller.rb

```diff
@@ -11,7 +11,7 @@ def show
       increment_step_counts
       analytics.idv_doc_auth_verify_visited(**analytics_arguments)
 
-      redirect_to failure_url(:fail) and return if any_throttled?
+      redirect_to throttled_url and return if any_throttled?
```

Comment:
Unless I am mistaken, we still need to add some throttle logic to either the `update` action or move this into a before action

Reply:
Thanks for catching that. I looked at adding a before action, noticed that we redirect to different error pages at different times because of throttles, and then got side tracked. Will follow up.

Reply:
On further looking at code, it looks like throughout the proofing flow, throttling doesn't get checked at the beginning of update/create actions. I wonder if we can address that as a bug separately from removing verify from FSM.

Reply:
Added a commit on a separate branch and we can decide if we want to pull it in to this branch. https://github.com/18F/identity-idp/commit/c5c474b326ea986aafcb26591a5655275adef83d

Reply:
I am totally fine with a different PR. We can open a bug ticket if we need. I do think we should fix it before deploying this.

Reply:
Opened a ticket. Checking that by "before deploying" you meant before LG-7402. I'd like to merge this for tomorrow because I have another branch that depends on it. https://cm-jira.usa.gov/browse/LG-8742